src/Symfony/Bridge/Twig/Extension/ImportMapExtension.php

```diff
@@ -23,6 +23,7 @@ public function getFunctions(): array
     {
         return [
             new TwigFunction('importmap', [ImportMapRuntime::class, 'importmap'], ['is_safe' => ['html']]),
+            new TwigFunction('importmap_link_tags', [ImportMapRuntime::class, 'linkTags'], ['is_safe' => ['html']]),
```

Comment:
what about making the `importmap()` helper also output those link tags and get rid of the new one?